python/notebook_utils.py

```diff
@@ -0,0 +1,702 @@
+#!/usr/bin/env python

```

Comment:
I dont think you have to copy whole file from notebook to this repo, some components are unnecessary.